lib/services/local/sessiontracker.go

```diff
@@ -123,7 +123,15 @@ func (s *sessionTracker) GetActiveSessionTrackers(ctx context.Context) ([]types.
 			return nil, trace.Wrap(err)
 		}
 
-		sessions[i] = session
+		if session.Expiry().After(time.Now().UTC()) {
+			sessions[i] = session
```

Comment:
I think we need to `append()` now. We also need to initialize `sessions` with len=0.